src/sass/components/_product-box.scss

```diff
@@ -80,3 +80,56 @@
     align-items: center;
   }
 }
+
+@media (min-width: 320px) and (max-width: 567px) and (orientation: portrait) {
+  .section--products {
+    .container {
+      .row {
+        .col-sm-12 {
```

Comment:
zmniejsz liczbę zagnieżdżeń

Reply:
Zmienione, złe przyzwyczajenie. Ale w nowych już na to uważam :)